### DISCUSSION TITLE
Copy db of magento 1.9.3.8 (maybe patched) to magento-lts

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1766] [Questions & Answers]
Just to make sure I do not have any stuff from magento left in my root. I wanted to do sort of a clean install from git magento-lts, but I would like to keep my configurations from the old magento db.

Would it work if I copy my old magento db, to a new db and use that db in the magento-lts install? Or is there an option to copy data from an old magento into a fresh installed magento-lts?

**Comment by justinbeaty** [comment]
@f1-outsourcing You can do a clean git clone with Magento LTS (or use composer) and just copy over the local.xml from the old site to the new site. All should work okay. This would transfer over all products, customers, orders, etc. I assume that's what you're interested in.

If there were any database updates in between your current Magento version and OpenMage, it will process those with the update scripts like any other Magento upgrade.

**Comment by f1-outsourcing** [comment]
I had some issues using the old db, so decided to stay with the new clean db.
